### PR TITLE
Add ban buttons to reddit live listing pages.

### PR DESCRIPTION
**Pull request by KeyserSosa**
As a general rule, CSS and JS I generate should be treated with suspicion.  (I'm certainly suspicious of it).  

screen shot 2016-04-04 at 3 38 03 pm

Tested these buttons (which have yes/no confirmation behavior) as well as the existing buttons on the permalink-analog pages. 

:eyeglasses: @spladug

### CODE REVIEW

File: reddit_liveupdate/public/static/js/liveupdate/report.js [hunk 1]
```diff
@@ -67,15 +67,34 @@
...
-      var url = '/api/live/' + r.config.liveupdate_event + '/' + action
+      confirm_admin(this.$adminButton, r.config.liveupdate_event).then(function() {
+        window.location.reload();
+      });
+    },
+  });
 
-      r.ajax({
+  var confirm_admin = function($btn, thing_id) {
+      console.log($btn);
```
Comment (spladug) [inline reddit_liveupdate/public/static/js/liveupdate/report.js]: :hocho:

File: reddit_liveupdate/public/static/js/liveupdate/report.js [hunk 2]
```diff
@@ -67,15 +67,34 @@
...
-    },
-  })
+      });
+  };
+
+  $('#siteTable .liveupdate-event button.admin').each(function() {
+    new r.ui.ConfirmButton({el: $(this)});
+  });
+  $('#siteTable .liveupdate-event .admin').on("confirm", function() {
+      var $this = $(this);
```
Comment (spladug) [inline reddit_liveupdate/public/static/js/liveupdate/report.js]: nit: indents should be two spaces

File: reddit_liveupdate/public/static/js/liveupdate/report.js [hunk 3]
```diff
@@ -67,15 +67,34 @@
...
-      var action = this.$adminButton.data('action')
-      var url = '/api/live/' + r.config.liveupdate_event + '/' + action
+      confirm_admin(this.$adminButton, r.config.liveupdate_event).then(function() {
+        window.location.reload();
+      });
+    },
+  });
 
-      r.ajax({
+  var confirm_admin = function($btn, thing_id) {
```
Comment (spladug) [inline reddit_liveupdate/public/static/js/liveupdate/report.js]: i'm a little confused by this function. what's it have to do with confirming admins?

File: reddit_liveupdate/public/static/js/liveupdate/report.js (hunk 1 above)
Comment (KeyserSosa) [inline reddit_liveupdate/public/static/js/liveupdate/report.js]: Whoops. I forgot to install my normal `pre-commit` hook locally.  Fixed that and I'll fix this.

File: reddit_liveupdate/public/static/js/liveupdate/report.js (hunk 3 above)
Comment (KeyserSosa) [inline reddit_liveupdate/public/static/js/liveupdate/report.js]: Naming is hard.  I'll change it to `confirm_admin_action`?

Comment (spladug) [inline reddit_liveupdate/public/static/js/liveupdate/report.js]: where's the confirming coming from? it's more just _doing_ it right?

Comment (KeyserSosa) [inline reddit_liveupdate/public/static/js/liveupdate/report.js]: `r.ui.ConfirmButton` triggers a `confirm` event on "yes".  It is fairly magical.  So how you feel about `post_admin_action`?

Comment (spladug) [inline reddit_liveupdate/public/static/js/liveupdate/report.js]: yeah, i just mean the confirmation has already happened at that point (this function is being called inside confirm event handlers) so it's more of just "post_admin_api" or something?

**Comment by KeyserSosa** [comment]
:haircut:

**Comment by spladug** [comment]
:fish:

**Pull request merged**